packages/libs/react-ui/src/components/Icons/IconWrapper.css.ts

```diff
@@ -5,6 +5,7 @@ import { style, styleVariants } from '@vanilla-extract/css';
 export const iconContainer = style([
   sprinkles({
     color: 'foreground',
+    display: 'flex',
```

Comment:
I removed this intentionally since the svg should be 100% width and height. Did you find an instance in which this is necessary?